hydromt_sfincs/sfincs.py

```diff
@@ -256,6 +257,8 @@ def setup_grid_from_region(
             Name of data source with basin (bounding box) geometries associated with
             the 'basins' layer of `hydrography_fn`. Only required if the `region` is
             based on a (sub)(inter)basins without a 'bounds' argument.
+        align : bool, optional
+            If True (default), align target transform to resolution.
```

Comment:
This only works for non-rotated right?

```suggestion
            If True (default), align target transform to resolution.
            Note that this has only been implemented for non-rotated grids.
```